### DISCUSSION TITLE
网页版调用外部播放器失败！！！

### DISCUSSION THREAD

**Discussion by kitkw** [AlistGo/alist#1515] [General]
之前正常！！一段时间没用现在一直是“服务器关闭”？？？web proxy 已开启  alist 也更新到最新了  还是不行

**Comment by kitkw** [comment]
potplayer

**Comment by HDqaq** [comment]
我也出现了 不知道怎么解决

**Reply by kitkw** [reply]
> 我也出现了 不知道怎么解决

我现在用的最新版 没有这问题   你更新试试看

**Comment by Ohcecho** [comment]
确实无法调用自带播放器，我的是vlm
